examples/standard/http-redirect-rewrite/gateway-redirect-http-https.yaml

```
@@ -0,0 +1,19 @@
+#$ Used in:
+#$ - site-src/guides/http-redirect-rewrite.md
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1
```

Tiny nit, most k8s examples put `apiVersion` before `kind`.
```suggestion
apiVersion: gateway.networking.k8s.io/v1
kind: Gateway
```

---

Updated!